composer.json

```
@@ -18,19 +18,18 @@
     },
     "require": {
         "php": "^7.2 || ^8.0",
-        "filp/whoops": "^2.5",
+        "filp/whoops": "^2.17",
         "middlewares/utils": "^3.0 || ^4.0",
         "psr/http-server-middleware": "^1.0",
         "psr/container": "^1.0 || ^2.0"
     },
     "require-dev": {
         "phpunit/phpunit": "^8|^9",
-        "laminas/laminas-diactoros": "^2.2",
-        "friendsofphp/php-cs-fixer": "^2.0",
-        "squizlabs/php_codesniffer": "^3.0",
-        "oscarotero/php-cs-fixer-config": "^1.0",
-        "eloquent/phony-phpunit": "^5.0 || ^7.0",
-        "phpstan/phpstan": "^0.12"
+        "laminas/laminas-diactoros": "^3",
+        "friendsofphp/php-cs-fixer": "^3",
+        "squizlabs/php_codesniffer": "^3.11",
+        "oscarotero/php-cs-fixer-config": "^1",
```

This is still failing. I think it need this:

```suggestion
        "oscarotero/php-cs-fixer-config": "^1 || ^2",
```

---

Yes! that and laminas-diactoros also. Thanks!

Feel free to merge the PR :+1: